### ISSUE TITLE
Bash auto complete does not display all possible tasks for my system.

### ISSUE THREAD

**Issue by MattSeen** [gruntjs/grunt-cli#80]
Hey,

Love this tool. It's really awesome and I appreciate the auto complete script you threw in for it. The only trouble is, I recently rewrote a build system from being a really, really big un-maintainable single grunt file to a modular system by using the plugin [load-grunt-config](https://github.com/firstandthird/load-grunt-config) and [jit-grunt](https://github.com/shootaroo/jit-grunt). These are great plugins for making a clearer a more maintainable build system as we grow it overtime. The only trouble is that breaks the assumption of the bash auto complete script.

[This is the line that causes my issue](https://github.com/gruntjs/grunt-cli/blob/master/completion/bash#L39)

~~~
# The current grunt version, available tasks, options, etc.
local gruntinfo="$(grunt --version --verbose 2>/dev/null)"
~~~

Now if this line were to read grunt --version --verbose --tasks <path to my tasks> I would be in business . As an aside, running grunt with a --tasks flag and not provide it an input causes an error on my system, but that is [probably a wider issue](https://github.com/gruntjs/grunt-cli/issues/45).

Basically I don't really know what would be the best approach to resolve this issue. You should probably provide a way for people to specify those additional parameters to the auto complete script. What that would look like I don't really know. Maybe an environment variable? Perhaps you allow me to have grunt run a script I would write and maintain, but then I can see other people having this issue in the future as well. And you know, open source or whatever.

I may have written myself into a corner on this because of the way those plugins work, but I'm pretty sure there could be some sort of solution for this. Any thoughts?
